__tests__/sources.ts

```diff
@@ -928,6 +929,38 @@ query Source($id: ID!) {
       'NOT_FOUND',
     ));
 
+  it('squad should have post moderation disabled', async () => {
+    await con.getRepository(SquadSource).save({
+      id: '12345',
+      name: 'Free For All',
+      handle: 'freeforall',
+      public: true,
+      moderationRequired: false,
+    });
+
+    const res = await client.query(QUERY, { variables: { id: '12345' } });
+    expect(res.errors).toBeFalsy();
+    expect(res.data.source.moderationRequired).toEqual(false);
+  });
+
+  it('squad should have post moderation enabled after update', async () => {
```

Comment:
This one should be inside editSquad and test that when passing moderationRequired it changes the field value in the db.